src/synth/fluid_mod.h

```diff
@@ -34,6 +34,7 @@ struct _fluid_mod_t
     unsigned char flags1;         /**< Source controller 1 flags */
     unsigned char src2;           /**< Source controller 2 */
     unsigned char flags2;         /**< Source controller 2 flags */
+    unsigned char trans;          /**< Output transform flag */
```

Comment:
For the sake of backward compatibility, this field must be initialized to zero in `new_fluid_mod()`.

Reply:
I did add `mod->trans = 0` in the function. Not sure if that's what you meant since other properties do not get the same treatment, but I didn't find any constructor for fluid_mod.

Reply:
You change is correct. The point here is that applications calling `new_fluid_mod` need to initialize the structure manually. This is common sense. Not initializing this new field would have required applications to change their code, i.e. would have been a breaking change.